include/experimental/__p0009_bits/default_accessor.hpp

```diff
@@ -42,12 +42,12 @@ struct default_accessor {
 
   MDSPAN_INLINE_FUNCTION
   constexpr data_handle_type
-  offset(data_handle_type p, size_t i) const noexcept {
+  offset(data_handle_type p, ptrdiff_t i) const noexcept {
```

Comment:
This does not conform with the C++ Standard, because `default_accessor` always takes the offset (in both `access` and `offset`) as `size_t`.  Offsets must always be nonnegative.

https://eel.is/c++draft/mdspan.accessor.default.overview